gobblin-core/src/main/java/gobblin/source/extractor/extract/restapi/RestApiExtractor.java

```diff
@@ -115,6 +115,15 @@ public void extractMetadata(String schema, String entity, WorkUnit workUnit)
       columnListInQuery = Utils.getColumnListFromQuery(inputQuery);
     }
 
+	// Deepak added the below code for building the exclusion list
+
+	String excludedColumns = workUnit.getProp(ConfigurationKeys.SOURCE_QUERYBASED_EXCLUDED_COLUMNS);
```

Comment:
Please format correctly. This line and the next line have different indentations.